vizro-core/docs/pages/user-guides/themes.md

```diff
@@ -10,7 +10,7 @@ you can still switch between the themes via the toggle button in the upper-right
 
 !!! example "Change theme"
     === "app.py"
-        ```py hl_lines="18"
```

Comment:
If we remove the code highlighting here, this example should change to a snippet code file (similar to how you've done it for the navigation), as the theme switch is only this one code of line `vm.Dashboard(pages=[page], theme="vizro_light")`.